packages/next/src/next-devtools/dev-overlay/components/errors/dev-tools-indicator/dev-tools-info/segments-explorer.tsx

```diff
@@ -0,0 +1,25 @@
+import { PageSegmentTree } from '../../../overview/segment-explorer'
+import { DevToolsInfo, type DevToolsInfoPropsCore } from './dev-tools-info'
+
+export function SegmentsExplorer({
+  routerType,
+  ...props
+}: DevToolsInfoPropsCore &
+  React.HTMLProps<HTMLDivElement> & {
+    routerType: 'app' | 'pages'
+  }) {
+  const isAppRouter = routerType === 'app'
+  return (
+    <DevToolsInfo title="Route Info" {...props}>
+      <div data-nextjs-segments-explorer>
+        <PageSegmentTree isAppRouter={isAppRouter} />
+      </div>
+    </DevToolsInfo>
+  )
+}
+
+export const SEGMENTS_EXPLORER_STYLES = `
+  [data-nextjs-segments-explorer] {
+    margin: -12px -8px;
```

Comment:
What are these values coupled with? Should be obvious on both sides what needs to change if values are adjusted. Though this does feel like it's masking a different issue.